vite.config.js

```diff
@@ -4,6 +4,7 @@ import svgr from "vite-plugin-svgr";
 import { resolve } from "path";
 import fs from "node:fs/promises";
 import { devServerHtmlScript } from "./scripts/dev-server-html-script.js";
+import { visualizer } from "rollup-plugin-visualizer";
```

Comment:
this runs any way on build? not just on demand? on purpose?

Reply:
@oryrosin You can also only run it on certain environments just add it to one of the switch cases in the function. I don't know what is your use for it (once in a while, every time etc)